style2.css

```diff
@@ -0,0 +1,539 @@
+@media screen and (max-width: 1680px) {
+  body {
+    width: 100%;
+    height: 100%;
+  }
+}
+
+@media screen and (max-width: 1440px) {
+  .main__pic {
+    margin: -100px 0 0 0;
+  }
+}
+
+@media screen and (max-width: 1283px) {
+  .article__charset {
+    margin: 30px 0 40px 0;
+  }
+
+  .article__info {
+    padding: 18px;
+  }
+
+  .buttons {
+    
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)